sdk/python/feast/infra/passthrough_provider.py

```diff
@@ -104,10 +104,14 @@ def offline_write_batch(
         self,
         config: RepoConfig,
         table: FeatureView,
-        data: pd.DataFrame,
+        data: pa.Table,
         progress: Optional[Callable[[int], Any]],
     ) -> None:
         set_usage_attribute("provider", self.__class__.__name__)
+
+        if "created" not in data.column_names:
```

Comment:
why is this necessary? I think most feature views don't have a created ts column, and it also might not be named `created`